keyboards/hhkb/yang/rules.mk

```diff
@@ -0,0 +1,31 @@
+# MCU name
+MCU = atmega32u4
+
+# MCU frequency
+F_CPU = 8000000
+
+# Bootloader selection
+BOOTLOADER = lufa-ms
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+
+LAYOUTS = 60_hhkb
+
+# Disable bluetooth until the UART code is merged
+# BLUETOOTH = AdafruitBLE_UART
+
+# Custom matrix file for the HHKB
+CUSTOM_MATRIX = lite
+SRC += matrix.c
+
+# Disable debounce
+DEBOUNCE = 0
```

Comment:
I believe this needs to be set in `config.h` (as `#define DEBOUNCE 0`) rather than here.

Reply:
^ Correct.  In the rules file, it won't actually do anything. 

Though, leaving it undefined will also set it to 0.